{{cookiecutter.project_name}}/.github/workflows/run-tests-fs.yml

```diff
@@ -0,0 +1,146 @@
+                    "features_transform_module": "pickup_features",
+                    "primary_keys": "zip"
+                  }
+                },
+                "existing_cluster_id": "0729-223229-zfjw22fr"
+              },
+              {
+                "task_key": "dropoff-features",
+                "notebook_task": {
+                  "notebook_path": "notebooks/GenerateAndWriteFeatures",
+                  "base_parameters": {
+                    "env": "staging",
+                    "test_mode": "True",
+                    "input_table_path": "/databricks-datasets/nyctaxi-with-zipcodes/subsampled",
+                    "timestamp_column": "tpep_dropoff_datetime",
+                    "output_table_name": "feature_store_taxi_example.trip_dropoff_features-test",
+                    "features_transform_module": "dropoff_features",
+                    "primary_keys": "zip"
+                  }
+                },
+                "existing_cluster_id": "0729-223229-zfjw22fr"
+              },
+              {
+                "task_key": "training",
+                "depends_on": [
+                  {
+                    "task_key": "dropoff-features"
+                  },
+                  {
+                    "task_key": "pickup-features"
+                  }
+                ],
+                "notebook_task": {
+                  "notebook_path": "notebooks/TrainWithFeatureStore",
+                  "base_parameters": {
+                    "env": "staging",
+                    "test_mode": "True",
+                    "training_data_path": "/databricks-datasets/nyctaxi-with-zipcodes/subsampled",
+                    "experiment_name": "{{cookiecutter.mlflow_experiment_parent_dir}}/{{cookiecutter.experiment_base_name}}-test",
+                    "model_name": "{{cookiecutter.model_name}}-test"
+                  }
+                },
+                "existing_cluster_id": "0729-223229-zfjw22fr"
+              }
+            ],
+            "git_source": {
+              "git_url": {% raw %}"${{ github.server_url }}/${{ github.repository }}",{% endraw %}
+              "git_provider": "gitHub",
+              "git_commit": {% raw %}"${{ github.event.pull_request.head.sha || github.sha }}"{% endraw %}
+            },
+            "access_control_list": [
+              {
+                "group_name": "users",
+                "permission_level": "CAN_VIEW"
+              }
+            ]
+            }`
+            return output.replace(/\r?\n|\r/g, '')    
+      - name: Feature Store/Model Training Integration Test
+        id: features-training-integration-test
+        run: |
+          python -m pip install --upgrade pip
+          pip install databricks-cli
+          databricks jobs configure --version=2.1
+          echo {% raw %}${{steps.integration-test-content.outputs.result}} > test.json {% endraw %}
+          databricks runs submit --json-file test.json --wait > tmp-output.json
```

Comment:
Isn't better if we chain a couple of `databricks/run-notebook@v0` together instead of in our stack the use of both the github action and the `databricks runs submit --wait` cli command.

Something more like this: 
```
 - name: Run first notebook
        uses: databricks/run-notebook@v0
        id: notebook-1
        with:
          local-notebook-path: path/to/first/notebooks_1.py
          ... // more fields to populate
          notebook-params-json: >
            {
              "input_1": "Hello",
              "input_2": "World!"
            }
          timeout-seconds: 1000
      - name: Run second notebook
        uses: databricks/run-notebook@v0
        id: notebook-2
        with:
          local-notebook-path: path/to/second/notebooks_2.py
          ... // more fields to populate
          notebook-params-json: >
            {
              "input_from_previous_notebook": "${{steps.notebook-1.outputs.notebook-output}}"
            }
          timeout-seconds: 1000
```

Reply:
Yeah, this option is fine for now, but would be good to prioritize the change @vladimirk-db  mentioned if we find that people are regularly working with this file - the GitHub workflow contents are a little tricky to modify as it stands (a combination of GHA parameter substitution, Javascript, and JSON contents)